numba/unicode.py

```diff
@@ -696,6 +696,68 @@ def zfill_impl(string, width):
     return zfill_impl
 
 
+# const style variable to enumerate variants of "strip" operations family
+UNICODE_OP_LSTRIP = 0
+UNICODE_OP_RSTRIP = 1
+UNICODE_OP_STRIP = 2
+
+
+@register_jitable
+def strip_impl(func_id, string, chars=' '):
```

Comment:
Default value for `chars` should be `None`.

Reply:
@esc for this line only or for all lines which contains `chars=' '` in this PR?

Reply:
Probably for all lines, for consistency.